From 01f6848ba0486d498e7977e258c665bdacf95cfc Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 19 May 2020 17:40:45 +0000
Subject: [PATCH 373/462] Mantis 0056312: Extend Desko-Penta Device-Handler by
 Accufill-SDK for reading US driving-licenses

for importing dll add AssureID\bin to include path

git-svn-id: svn://localhost/SelfServiceBuild/trunk@683 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index 6b9dc0c..c3b386c 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -394,9 +394,9 @@ set DIR_SPEECH11=%DIR_GLOBALSLIB%\Microsoft_SDKs\Speech\v11.0
 set INC_SPEECH11=-I%DIR_SPEECH11%\Include
 set LIB_SPEECH11=%DIR_SPEECH11%\Lib\sapi.lib
 
-set DIR_ASSUREID=%DIR_GLOBALSLIB%\AssureID\5.2.61.40\sdk
-set INC_ASSUREID=-I%DIR_ASSUREID%\com -I%DIR_ASSUREID%\dotnet\4.0
-set LIB_ASSUREID=%DIR_ASSUREID%\dotnet\4.0\x86\AssureTec.AssureID.Platform.SDK.Extensions.lib
+set DIR_ASSUREID=%DIR_GLOBALSLIB%\AssureID\5.2.61.40
+set INC_ASSUREID=-I%DIR_ASSUREID%\sdk\com -I%DIR_ASSUREID%\sdk\dotnet\4.0 -I%DIR_ASSUREID%\bin
+set LIB_ASSUREID=%DIR_ASSUREID%\sdk\dotnet\4.0\x86\AssureTec.AssureID.Platform.SDK.Extensions.lib
 
 set DIR_QTSDK=%DIR_GLOBALSLIB%\qtsdk\6.0\win32
 set INC_QTSDK=-I%DIR_QTSDK%\cincludes
-- 
2.41.0.windows.1

